src/Database/Connection.php

```diff
@@ -187,7 +187,19 @@ protected function createDrivers(array $config): array
     public function __destruct()
     {
         if ($this->_transactionStarted && class_exists(Log::class)) {
-            Log::warning('The connection is going to be closed but there is an active transaction.');
+            $message = 'The connection is going to be closed but there is an active transaction.';
+
+            $requestUrl = $_SERVER['REQUEST_URI'] ?? '';
+            if (!empty($requestUrl)) {
```

Comment:
```suggestion
            if ($requestUrl) {
```
always suffices for existing vars.